### PR TITLE
Adds Captains Space Helmet, Makes the Captain's Hat and Gas Mask No Longer Space Helmets

### PR DESCRIPTION
**Pull request by Flaborized**
[BALANCE] [INPUT WANTED]

## About the PR 

Adds in the captain's space helmet, meant to match up with the captain's space suit. It spawns in the captain's locker. (to make it more -Exclusive- to the cap) It also comes in blue with the medal reward and has a red version for admins to spawn, just like all the other cap stuff. 

This PR also makes the captain's hat and regular gas masks no longer count as spacewear, so you no longer get 100% space immunity from wearing one of those + a spacesuit of some sort.

## Why's this needed? 

It always bugged me that the captain's spacesuit didn't have a helmet to go with it, I think the full suit look is Cool and Good, and having a full set really completes the suit. Literally I guess.

As for the hat and mask, if you look closely at either of them you'll notice that they don't protect your head from the frosty void of space much at all! Aside from not *looking* like spacewear, they also didn't communicate that they were spacewear well at all: the gas mask description says it'd only give you 7% cold immunity (I think that's a pretty fair number) but if you paired it with a suit, it'd count it as a full set and jump that number up to 100% instantly. Very strange behavior for items not indicated to be spacewear that also don't visibly seal your head in and insulate you at all.

Also they're both fairly fashionable pieces of apparel, and the mask in particular lets you maintain a lot of your aesthetic by letting you wear some fancy hat item. I think this is actually very silly; spacesuits being hazardous gear and looking big and clunky is part of their whole aesthetic, and I feel like if the knowledge was more widespread, people wouldn't tend to wear space helmets at all as they're less fashionable and less optimal than gas masks or the cap hat. Plus to reiterate, them being spaceproof is aesthetically inconsistent imo and that's not as good as having it make sense in this case imo. Functionality doesn't match what the thing is in this case, this fixes that.

## Changelog

```changelog
(u)Flaborized
(*)Added the captain's space helmet to their locker. Captain's hats and gas masks no longer count as space helmets.
```

### CODE REVIEW

**Comment by DimWhat** [comment]
Always thought it was kinda weird to wear a hat and a space suit.

**Comment by frawhst** [comment]
Making the gas mask not spaceproof is a pretty notable balance change, since now the only spaceproof mask is the vape-o-matic. Means you can't be spaceproof with just a mask and a helmet anymore unless you grab one of the obscure jumpsuits that does that. I see a lot of people use a gas mask and a space helmet as cold protection, and that won't work after this gets merged.

Not saying this is a bad thing, just think this should be marked as a balance change as well.

**Comment by Kyle2143** [comment]
I like the new sprites, and making gas masks not be spacewear cause it doesn't really make sense for them to be spacewear and them being spacewear isn't even a joke. But I always liked the Captain's hat to be considered spacewear, that was just funny basically cause it made no sense. I wouldn't really like to remove that.

**Comment by MatheusIsNotAvailable** [comment]
Sprites are cool. But I am not a fan on the balance changes, I understand why, but I would rather not have to need this extra thing, could just make it so it requires both a mask and a hat instead of just a mask, also, most people use space helmets because of their stats not for cold protection. I feel like a lot of people would just default to always wearing space helmets after this, wich is not What I want to see, since it obscures hair making people look less interesting to look at. Hats are also a lot of the time one of the big ways wich we convey special roles, see commander hat or HoS beret, if you see someone with a hos hat floating around in space suit you KNOW they are the HoS. if this were to get merged it would just be a bunch of spaceman in spacesuits that look all the same.

**Comment by Flaborized** [comment]
> Not saying this is a bad thing, just think this should be marked as a balance change as well.

Good point. Done!


> I like the new sprites, and making gas masks not be spacewear cause it doesn't really make sense for them to be spacewear and them being spacewear isn't even a joke. But I always liked the Captain's hat to be considered spacewear, that was just funny basically cause it made no sense. I wouldn't really like to remove that.

I don't actually think there'd be any point to wearing the captain's space helmet if the captain's armored (but not visually) space helmet (but not visually) outclassed it in both Usefulness and Style. I feel like an item being spacewear because It Makes No Sense isn't really much of a joke, I always just assumed super old SS13 devs just... didn't make a captain's helmet. Either way I think leaving the cap's hat as a space helmet is incongruent with giving the captain a space helmet item.



> Sprites are cool. But I am not a fan on the balance changes, I understand why, but I would rather not have to need this extra thing, could just make it so it requires both a mask and a hat instead of just a mask, also, most people use space helmets because of their stats not for cold protection. I feel like a lot of people would just default to always wearing space helmets after this, wich is not What I want to see, since it obscures hair making people look less interesting to look at. Hats are also a lot of the time one of the big ways wich we convey special roles, see commander hat or HoS beret, if you see someone with a hos hat floating around in a space suit you KNOW they are the HoS. if this were to get merged it would just be a bunch of spaceman in spacesuits that look all the same.

Hi not to be rude, but I frankly think this is just Wrong. As far as I can tell, most players are *under the impression* that space helmets are needed to resist the cold vacuum of space (because that's what we've visually communicated, gas masks working is semi-obscure, or at least something you need to be pretty experienced to know about), and you don't see everyone who thinks that you need space helmets walking around with them on all the time, totally giving up on fashion or armor. I see absolutely no world where people (consistently) give up wearing fashion items like the HoS beret or *whatever hat* solely for the sake of space helmets, if they wouldn't already have done it before this change.

And *even if they did* during breaches or spacewalks, I think that's Good Actually. It Does make people look less unique or identifiable, but that's the entire aesthetic of a space suit. 'Unidentified human in a spacesuit, dead, floating in space' is a common science fiction visual trope, and that's because it's a pretty cool way to portray spacesuits. Situations where the whole station is breached and everyone is suited up with their face hidden and their fashion gone is actually *cool* and good, it's both a unique visual and mechanical situation.

Even so, spacesuits and helmets are *decently* limited in their availability anyways. The resource scarcity there is intentional, and having a 'cheat' item like gas masks that are very common defeats the point of having a limited supply of space helmets. It's not exactly what is materially 'good' for players (in that they don't get what they want In The Game, 'To Win') but I believe this is change is good for the game.

**Review by mordent-goonstation — APPROVED** [review]

File: code/WorkInProgress/rewardsLocker.dm [hunk 1]
```diff
@@ -622,6 +622,15 @@
 					H.set_clothing_icon_dirty()
 					succ = TRUE
 
+				else if (istype(M, /obj/item/clothing/head/helmet/space/captain))
+					var/prev = M.name
+					M.name = "\improper commander's space helmet"
```
Comment (mordent-goonstation) [inline code/WorkInProgress/rewardsLocker.dm:627]: Use of `\improper` here might not quite be correct. "a commander's space helmet" seems fine to me?

**Comment by Camryn-Buttes** [comment]
Extremely firm disagreement here.

Gas masks being spaceworthy is, in my experience, already common knowledge.

Space helmets already have a pretty big advantage over most other headgear (relatively high protection + they function as a welding mask for everything except flashes)

I cannot, in good conscience, accept the precedent this PR sets. The massive hit to self-expression is not worth the minor gains to visual consistency.

Hats are a core component of spaceperson customization and this change severely hampers our freedom to wear our preferred headgear in space.

**Review by virvatuli-gh — APPROVED** [review]

File: code/WorkInProgress/rewardsLocker.dm (hunk 1 above)
Comment (Flaborized) [reply code/WorkInProgress/rewardsLocker.dm]: good idea, axed the \improper

**Pull request merged**